org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT » test {yaml=reference/ilm/start-stop-ilm/line_56} fails with expecting STOPPING but was STOPPED

https://gradle-enterprise.elastic.co/s/qz63x47f55v2m/tests/jbm2x6tnwtxku-ov7tbnxyfm6qe

```
:docs:integTestRunnerorg.elasticsearch.smoketest.DocsClientYamlTestSuiteIT » test {yaml=reference/ilm/start-stop-ilm/line_56} (0.480s)
java.lang.AssertionError: 
Failure at [reference/ilm/start-stop-ilm:78]: $body didn't match expected value:
                         $body: 
                  operation_mode: expected String [STOPPING] but was String [STOPPED]

Caused by: java.lang.AssertionError: 
$body didn't match expected value:
                         $body: 
                  operation_mode: expected String [STOPPING] but was String [STOPPED]

```

Reproduction line:
```
./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/ilm/start-stop-ilm/line_56}" -Dtests.seed=3D5129E7020BB3D4 -Dtests.security.manager=true -Dtests.locale=zh-HK -Dtests.timezone=America/Thule -Dcompiler.java=12 -Druntime.java=11
```

--- a/docs/reference/ilm/start-stop-ilm.asciidoc
+++ b/docs/reference/ilm/start-stop-ilm.asciidoc
@@ -107,7 +107,7 @@ GET _ilm/status
   "operation_mode": "STOPPING"
 }
 --------------------------------------------------
-// TESTRESPONSE[skip:"AwaitsFix https://github.com/elastic/elasticsearch/issues/46528"]
+// TESTRESPONSE[s/"STOPPING"/$body.operation_mode/]
 
 The ILM service will then, asynchronously, run all policies to a point
 where it is safe to stop. After ILM verifies that it is safe, it will
@@ -129,6 +129,7 @@ GET _ilm/status
   "operation_mode": "STOPPED"
 }
 --------------------------------------------------
+// TESTRESPONSE[s/"STOPPED"/$body.operation_mode/]
 
 [float]
 === Starting ILM

